app/javascript/components/get-version.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Query } from 'react-apollo';
+import gql from 'graphql-tag';
+
+const getVersion = gql`
+  {
+    schemaSha
+  }
+`;
+
+let lastValue = null;
+
+export const Version = () => (
+  <Query query={getVersion}>
```

Comment:
Is there a reason lastValue is defined outside of the Version component? Does it make it such that new renders of the Version component don't reset lastValue?

Reply:
Yes, exactly. This is a global variable, which is outside the scope of the Version component. The reason `lastValue` is a global variable is to have easy access to the current SHA value on the client. It is initialized as `null`, but is reset to the current SHA after the first instance of Version.